models.go

```diff
@@ -0,0 +1,403 @@
+package configcat
```

Comment:
low: It would be great to synchronize the intellisense docs in this file with [those of the reference implementation](https://github.com/configcat/.net-sdk/blob/master/src/ConfigCatClient/Models/Config.cs#L16), which was reviewed & approved by the UX team (that, is by you :D)

Reply:
Sure, I worked mostly based on that, however I had to improvise some parts as the reference implementation doesn't cover all fields (not everything is available publicly available). Also, I had to phrase sentences as the first word of the comment must be the name of the thing that I describe.

I've added the missing parts (e.g., comparators) and fixed some others.